practice-one/src/components/menu.html

```diff
@@ -0,0 +1,13 @@
+<section class="menu-container">
+  <h2 class="menu-heading"><span class="primary-color">Menu</span> That <span class="accent-color">Always</span> Make You Fall In <span class="primary-color">Love</span></h2>
+  <div class="menu-tag">
+    <div class="menu-change-button">Ramen</div>
+    <div class="menu-change-button">Breakfast</div>
+    <div class="menu-change-button">Lunch</div>
+    <div class="menu-change-button">Dinner</div>
+    <div class="menu-change-button">Mexican</div>
+    <div class="menu-change-button">Italian</div>
+    <div class="menu-change-button">Desserts</div>
+    <div class="menu-change-button">Drinks</div>
```

Comment:
Don't repeat yourself, create component for this

Reply:
and also check the html tag

Reply:
Thanks , i fixed it